archinstall/lib/models/dataclasses.py

```diff
@@ -14,38 +15,60 @@ def parse_version(self) -> List[str]:
 
 		return versions
 
-	@classmethod
 	def major(self) -> str:
 		return self.parse_version()[0]
 
-	@classmethod
 	def minor(self) -> str:
 		versions = self.parse_version()
 		if len(versions) >= 2:
 			return versions[1]
 
-	@classmethod
+		return ""
```

Comment:
I've only glanced over the code, and I honestly can't remember my own logic for this one. But I suspect we'd want to return `"0"` where version definitions are not found, as we'd want to convert this to something comparable?
I guess `""` is comparable as well, but something inside feels better comparing against numbers.

Reply:
Good point, I've changed the code to reflect your critique